frontend/.env.example

```
@@ -1 +1,2 @@
-VITE_IP_ADD=YOUR_IP:PORT
\ No newline at end of file
+VITE_IP_ADD=YOUR_IP
+VITE_API_BASE_URL=losthost:port
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)